orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/EqualTo.feature

```diff
@@ -475,1103 +131,118 @@ Feature: User can specify that a value is equalTo a required value
       | foo |
       | 15  |
 
-  Scenario: 'EqualTo' a value and must be null should emit null
+  Scenario: 'EqualTo' a value and must be null should be contradictory
     Given there is a field foo
     And foo is equal to "a"
     And foo is null
-    Then the following data should be generated:
-      | foo  |
-      | null |
+    Then no data is created
 
 ### ofType ###
 
-  Scenario: 'OfType' string 'equalTo' a string value should be successful
-    Given there is a field foo
-    And foo is equal to "0123456789"
-    And foo is of type "string"
-    Then the following data should be generated:
-      | foo          |
-      | null         |
-      | "0123456789" |
-
-
-  Scenario: 'EqualTo' an integer with 'ofType' integer is successful
+  Scenario: 'EqualTo' should combine with types
```

Comment:
Since this is titled `with types` not `with of type integer` it might as well be made a scenario outline test with all the types

Reply:
These should either be kept seperate and titled (e.g.) ``EqualTo` should combine with type "string"` or combined into one single [scenario outline](https://cucumber.io/docs/gherkin/reference/#scenario-outline). I don't think we should have multiple scenarios with the same title